modules/openshift-developer-cli-setting-and-unsetting-environment-variables.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/managing-environment-variables-in-odo.adoc
+
+[id="setting-and-unsetting-environment-variables._{context}"]
+
+= Setting and unsetting environment variables
+
+.Procedure
+
+* To set an environment variable in a component:
++
+----
+$ odo config set --env VARIABLE=value
```

Comment:
If "VARIABLE" in these examples is a replaceable value, it should be <variable> according to our doc guidelines. So:

```
 $ odo config set --env <variable>=<value>
```